src/Integration.Vsix/CFamily/PortedFromJava/MsvcDriver.cs

```diff
@@ -273,7 +269,7 @@ public static Request ToRequest(CFamilyHelper.Capture[] captures)
                     }
                     else
                     {
-                        var relativeOrAbsolutePath = arg.Replace("\"", "");
+                        var relativeOrAbsolutePath = RemoveDoubleQuotes(arg);
```

Comment:
you should do the same `RemoveDoubleQuotes ` on  `dirStr` before calling absolute in case of the `/I` option